admission/hooks/validate_networkpolicy.go

```diff
@@ -0,0 +1,86 @@
+package hooks
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strconv"
+
+	calicov3 "github.com/projectcalico/libcalico-go/lib/apis/v3"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
+	"sigs.k8s.io/controller-runtime/pkg/webhook"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+)
+
+const (
+	annMinimumPolicyOrder = "admission.cybozu.com/min-policy-order"
+)
+
+// cnpvlog is for logging in this package.
+var cnpvlog = logf.Log.WithName("calico-networkpolicy-validator")
+
+// +kubebuilder:webhook:verbs=create;update,path=/validate-projectcalico-org-networkpolicy,mutating=false,failurePolicy=fail,groups=crd.projectcalico.org,resources=networkpolicies,versions=v1,name=vnetworkpolicy.kb.io
+// +kubebuilder:rbac:groups="",resources=namespaces,verbs=get;list;watch
```

Comment:
Is it necessary to add permission to access to NetworkPolicy?

Reply:
No, because the webhook does not send requests for NetworkPolicy to API server.

Reply:
I see.